Vegetarian-Food-Finder/Api/ApiManger.swift

```diff
@@ -2,8 +2,35 @@
 //  ApiManger.swift
 //  Vegetarian-Food-Finder
 //
-//  Created by Abdullahi Ahmed on 7/6/22.
+//  Created by Abdullahi Ahmed on 7/7/22.
 //
 
 import UIKit
 
+final class ApiManger {
+    
+    private class ApiConstant {
```

Comment:
if the constants are being used internally only, you don't need to wrap it in another private class. Just declare the constants directly into `ApiManager`.